.env.example

```diff
@@ -1,12 +1,25 @@
 DATABASE_URL=postgresql://localhost/carbon
 REDIS_URL=redis://127.0.0.1:6379
+
 ETHEREUM_RPC_ENDPOINT=xxxxxxxxxx
-SHOULD_HARVEST=1
-SHOULD_POLL_QUOTES=1
-POLL_QUOTES_INTERVAL=30000
+SEI_RPC_ENDPOINT=xxxxxxxxxx
```

Comment:
As this serves also licensees let's clarify that the chain specific keys are optional and are not required if the specific chain isn't relevant for the deployment. Same for the price APIs, not all of them are needed for licensees who deploy to specific chain.

Reply:
Cool, for now I just added some comments to .env.example for this.
Later we can improve this to validate env variables according to the deployments configured. 
e.g: codex api key is required when there is a network other than ethereum => validate this and throw an error if needed